src/xrGame/player_hud.cpp

```diff
@@ -53,16 +53,20 @@ void player_hud_motion_container::load(IKinematicsAnimated* model, const shared_
             {
                 pm.m_base_name = anm;
                 pm.m_additional_name = anm;
+                pm.m_anim_speed = 1.f;
             }
             else
             {
-                R_ASSERT2(_GetItemCount(anm.c_str()) == 2, anm.c_str());
+                R_ASSERT2(_GetItemCount(anm.c_str()) <= 3, anm.c_str());
                 string512 str_item;
                 _GetItem(anm.c_str(), 0, str_item);
                 pm.m_base_name = str_item;
 
                 _GetItem(anm.c_str(), 1, str_item);
-                pm.m_additional_name = str_item;
+                pm.m_additional_name = (strlen(str_item) > 0) ? pm.m_additional_name = str_item : pm.m_base_name;
```

Comment:
The code `pm.m_additional_name = pm.m_additional_name = str_item` looks bad.
I believe, it should be this:
```suggestion
                pm.m_additional_name = xr_strlen(str_item) > 0 ? str_item : pm.m_base_name;
```

Reply:
Yeah, copy-pasted it with minimal changes. Not sure about ret value from assignment operation